gallery/src/components/NxFontAwesomeIcon/NxFontAwesomeIconPage.tsx

```diff
@@ -33,12 +33,26 @@ const NxFontAwesomeIconPage = () => {
           <a href="https://github.com/FortAwesome/react-fontawesome#features" target="_blank">documentation</a>
           {' '}for details on available props
         </p>
+        <h3 className="nx-h3">Accessibility</h3>
+        <p className="nx-p">
+          By default Font Awesome icons are set to <code className="nx-code">aria-hidden="true"</code> which means that
+          they are not "seen" by screen readers. Normally this is acceptable because icons are usually presentational
+          in nature, however if the icon has information that is important to convey to the user (e.g. an icon only
+          button) the value of <code className="nx-code">aria-hidden</code> should be set to
+          {' '}<code className="nx-code">false</code> and an <code className="nx-code">aria-label</code> with
+          explanatory text should also be employed. The last two examples demonstrate this.
+        </p>
+        <p className="nx-p">
+          If you are in doubt about whether an icon should be made accessible consider whether the user could perform
+          their given task, or understand an explaination if that icon was not there.
```

Comment:
May want to also point out the converse: that if adding a label to the button would result in duplicitous announcements (e.g. due to adjacent text content) then the label should not be added.

Reply:
> May want to also point out the converse: that if adding a label to the button would result in duplicitous announcements (e.g. due to adjacent text content) then the label should not be added.

I felt like that was implied, but I have added some (hopefully) clarifying text here: https://github.com/sonatype/sonatype-react-shared-components/pull/222/commits/a905b20509beab7ef707a886a7e134983c514869